### PR TITLE
Military time option.

### PR DESCRIPTION
**Pull request by KaeganClark**
Currently when you turn on the digital clock it will always display in military time format. I made the clock default to standard 12-hour format with the option to use military time.

If you are hard set on military time being the default I can make 12-hour format the extra option instead.

Let me know if you have any questions.

### CODE REVIEW

File: lib/analog_clock.dart [hunk 1]
```diff
@@ -29,6 +30,7 @@ class AnalogClock extends StatefulWidget {
       this.showNumbers = true,
       this.showSecondHand = true,
       this.showAllNumbers = false,
+      this.useMilitaryTime = false,
```
Comment (furkantektas) [inline lib/analog_clock.dart]: To be able to backward compatible, I think `useMilitaryTime` should be true by default.

File: lib/analog_clock.dart [hunk 2]
```diff
@@ -51,6 +53,7 @@ class AnalogClock extends StatefulWidget {
       showNumbers = true,
       showAllNumbers = false,
       showSecondHand = true,
+      useMilitaryTime = false,
```
Comment (furkantektas) [inline lib/analog_clock.dart]: Also I think this should be `true` by default.

**Comment by furkantektas** [comment]
Hi @KaeganClark, this is indeed a useful feature, thank you :) 

I think it would be convenient to add 'AM/PM' indicator as well and keep military time format by default to be backward compatible. What do you think?

File: lib/analog_clock.dart (hunk 1 above)
Comment (KaeganClark) [reply lib/analog_clock.dart]: I have submitted the change to make it backwards compatible.

File: lib/analog_clock.dart (hunk 2 above)
Comment (KaeganClark) [reply lib/analog_clock.dart]: Done!

**Comment by KaeganClark** [comment]
Hey @furkantektas, thank you for considering my changes. 

I have made it backwards compatible as you requested. I also added the meridiem to the displayed time. That was a great addition.

Let me know if you have any more conflicts.

**Comment by furkantektas** [comment]
Hi @KaeganClark,
My sincere apologies for not responding this in a timely manner. I really appreciate your help and contributions. I could only test the changes today, and merge it to the master. 
Thanks you for all your help and effort.

**Pull request merged**